packages/core/webpack/config/loaders.js

```diff
@@ -127,6 +127,11 @@ module.exports = props => {
     {
       test: /\.(scss|sass|css)$/i,
       use: !props.extract ? ["iso-morphic-style-loader"].concat(cssLoaders) : [MiniCssExtractPlugin.loader].concat(cssLoaders)
+    },
+    {
+      test: /\.worker\.js$/,
+      loader: "worker-loader",
+      options: {inline: true, fallback: false, publicPath: "/src/actions/"}
```

Comment:
have you tested a prod build? I see this `publicPath` being relative and get paranoid 😅

Reply:
Yes, I did. Haha, same here.